sdk/Sdk.Generators/Constants.cs

```diff
@@ -5,6 +5,12 @@ namespace Microsoft.Azure.Functions.Worker.Sdk.Generators
 {
     internal static class Constants
     {
+        // MSBuild Props
+        internal const string EnableSourceGenProp = "build_property.FunctionsMetadataSourceGen_Enabled";
```

Comment:
Small suggestion to consider: I like using nested classes to organize my constants as the file gets bigger. Something like:

``` CSharp
internal static class Constants
{
    public static class BuildProperties
    {
        public const string SourceGenEnabled = "build_property.FunctionsMetadataSourceGen_Enabled";
    }

    public static class Types
    {
        public const string BindingAttribute = "Microsoft.Azure.Functions.Worker.Extensions.Abstractions.BindingAttribute";
        public const string OutputBindingAttribute = "Microsoft.Azure.Functions.Worker.Extensions.Abstractions.OutputBindingAttribute";
    }
}
```

But of course, this is just how I like to do it.